### ISSUE TITLE
Edge rendering issues

### ISSUE THREAD

**Issue by MorrisJobke** [nextcloud/server#12291]
The current web UI (master, but stable14 had similar issues) looks not that good in Edge:

<img width="1136" alt="bildschirmfoto 2018-11-05 um 18 10 38" src="https://user-images.githubusercontent.com/245432/48014112-3f314300-e126-11e8-91aa-54b366cc3408.png">

cc @nextcloud/designers

**Comment by jancborchardt** [comment]
@skjnldsv so grid view doesn’t work in Edge either? Should we also do the "Only list view" fix there?

**Comment by skjnldsv** [comment]
@jancborchardt nope, edge is really decent on feature support. I'm guessing something needs a fix, but it should be ok :)

**Comment by rullzer** [comment]
Looks fine to me. We shall call it the windows experience :)

**Comment by skjnldsv** [comment]
Works fine on latest edge :thinking: 
![capture d ecran_2018-11-08_18-06-45](https://user-images.githubusercontent.com/14975046/48214825-16f15080-e381-11e8-9b79-61c74c49f15b.png)

**Comment by skjnldsv** [comment]
@MorrisJobke any updates?
